tests/test_snapshot.py

```diff
@@ -473,3 +478,48 @@ def test_pc_rating_rate_none(user: UserClient):
 def test_pc_2(user: UserClient):
     s = file('laptop-hp_255_g3_notebook-hewlett-packard-cnd52270fw.snapshot')
     snapshot, _ = user.post(res=Snapshot, data=s)
+
+
+@pytest.mark.mvp
+def test_save_snapshot_in_file():
+    """ This test check if works the function save_snapshot_in_file """
+    snapshot_no_hid = file('basic.snapshot.nohid')
+    save_json(snapshot_no_hid)
+
+    uuid = snapshot_no_hid['uuid']
+    files = [x for x in os.listdir(TMP_SNAPSHOTS) if uuid in x]
+
+    snapshot = {'software': '', 'version': '', 'uuid': ''}
+    if files:
+        path_snapshot = "{dir}/{file}".format(dir=TMP_SNAPSHOTS, file=files[0])
+        file_snapshot = open(path_snapshot)
+        snapshot = json.loads(file_snapshot.read())
+        file_snapshot.close()
```

Comment:
Maybe this 3 lines could be refactored using `with open(....):` [with statement](https://docs.python.org/3/tutorial/inputoutput.html#reading-and-writing-files). That automatically closes the file.